lsr_role2collection/collection_readme.txt

```diff
@@ -0,0 +1,47 @@
+﻿# Linux System Roles Ansible Collection
+Ansible collection for interacting with the Linux System Roles.
+
+## Currently supported distributions
+* Fedora
+* Red Hat Enterprise Linux (RHEL 6+)
+* RHEL 6+ derivatives such as CentOS 6+
+
+NOTE: Some roles are not supported in RHEL6 and RHEL7. For more details about the individual roles you are interested in, see the documentation.
+
+## Support
+
+### Supported Ansible Versions
+The supported Ansible versions are aligned with currently maintained Ansible versions that support Collections (Ansible 2.9 and later). You can find the list of maintained Ansible versions [here](https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#release-status).
+

```

Comment:
Shouldn't we say something like "the modules and other plugins contained in this collections are considered internal unless explicitly noted, the only public interfaces are those of the roles, as described in their respective READMEs" ?

Reply:
yes - how about "The modules and other plugins in this collection are private, used only internally to the collection, unless otherwise noted."